docs/2.0/docs/accountfactory/tutorials/remove-account.md

```diff
@@ -43,54 +44,55 @@ All the code changes recommended here should be committed for each repository, *
   terragrunt plan # Verify no accounts are set for destruction but outputs are updated
   terragrunt apply # Apply the changes.
   ```
-1. (Optional) Remove AWS Transit Gateway(TGW) attachments. If an account being removed uses AWS TGW, you will need to remove the attachments by running the following commands sequentially:
+7. (Optional)  Remove AWS Transit Gateway (TGW) attachments if the account uses AWS TGW by running:
 
     ```bash
     cd <NETWORK-ACCOUNT>/_global/transit_gateway_attachments/
     terragrunt plan
     terragrunt apply
     ```
 
```

Comment:
```suggestion
This is a plan/apply operation because the TGW module reads the `accounts.yml` file to determine which accounts are attached to the TGW.
```